examples/advanced/reconstruct_python.py

```diff
@@ -8,10 +8,15 @@
 
 """
 
-from lark import Token
+from lark import Lark, Token
 from lark.reconstruct import Reconstructor
 
-from python_parser import python_parser3
+from python_parser import PythonIndenter
```

Comment:
If we have the python grammar in the lark stdlib, maybe add an compatible PythonIndenter in `lark`?

Reply:
Good point! Maybe in lark/indenter.py

Reply:
Yeah.